appGeo/src/app/game/game.component.ts

```diff
@@ -73,6 +80,12 @@ export class Game implements OnInit {
         this.players.forEach((player: Player, key: number) => {
           databaseEventListener(player.getDatabasePath(), this.updatePlayerDatabase.bind(this));
         });
+        databaseEventListener(GAMERULE_PATH, this.updateGameRulesDatabase.bind(this));
+        databaseEventListener(MAP_PATH, this.updateMapDatabase.bind(this));
+        databaseEventListener(START_TIME_PATH, this.updateStartTime.bind(this));
+        databaseEventListener(END_TIME_PATH, this.updateEndTime.bind(this));
+        databaseEventListener(STATUS_PATH, this.updateStatus.bind(this));
+
```

Comment:
Not sure why we are adding listeners in the forEach loop for each player when seemingly these listeners aren't even player specific

Reply:
These aren't in the forEach loop? These are all after.

Reply:
my bad LOL I must be out of my mind right now